package/src/values/selector.ts

```diff
@@ -0,0 +1,24 @@
+import type { SkiaValue } from "./types";
+
+export type SelectorType<TReturn, TInput = unknown> = {
```

Comment:
Should it be called SkiaSelector?
Should we create a `SkiaReadonlyValue<T> = SkiaValue<T> | SkiaSelector<T>`? (or have as a property type of compositive component that currently accept `SkiaValue`.